packages/components/controlled-input/src/ControlledInput.tsx

```diff
@@ -32,37 +34,36 @@ export interface ControlledInputProps extends HTMLProps<HTMLInputElement> {
   indeterminate?: boolean;
 }
 
-export const ControlledInput = ({
-  checked,
-  className,
-  disabled = false,
-  id,
-  labelText,
-  name,
-  onBlur,
-  onChange,
-  onFocus,
-  required = false,
-  testId = 'cf-ui-controlled-input',
-  type = 'checkbox',
-  value,
-  willBlurOnEsc = true,
-  indeterminate,
-  ...otherProps
-}: ControlledInputProps) => {
+const _ControlledInput = (
```

Comment:
Is the underscore the proper way to mark module scoped vars?

Reply:
I copied that style from Badge and didn't see any other. Could also call it ControlledInputWithoutRef but I like it like that :)